arch/risc-v/src/common/riscv_mmu.h

```diff
@@ -90,7 +94,7 @@
 
 #ifdef CONFIG_ARCH_MMU_TYPE_SV39
 #define RV_MMU_PTE_PADDR_SHIFT  (10)
-#define RV_MMU_PTE_PPN_MASK     ((1 << RV_MMU_PTE_PADDR_SHIFT) - 1)
+#define RV_MMU_PTE_PPN_MASK     (((1ul << 44) - 1) << RV_MMU_PTE_PADDR_SHIFT)
```

Comment:
```suggestion
#define RV_MMU_PTE_PPN_MASK     (((UINT64_C(1) << 44) - 1) << RV_MMU_PTE_PADDR_SHIFT)
```

Reply:
They are applicable for `CONFIG_ARCH_MMU_TYPE_SV39`